core/src/Streamly/Internal/Data/Stream/Serial.hs

```diff
@@ -72,15 +72,15 @@ type Serial = SerialT IO
 -- Generate an infinite stream by repeating a pure value.
 --
 {-# INLINE_NORMAL repeat #-}
-repeat :: Monad m => a -> Stream m a
+repeat :: Monad m => a -> SerialT m a
```

Comment:
Now that we have `repeat` in the Data.Stream module we can just directly export that from this module instead of redefining it here.